drivers/led/lp5562.c

```diff
@@ -978,28 +1030,26 @@ static const struct led_driver_api lp5562_led_api = {
-		"Red channel current must be between 0 and 25.5 mA.");	\
-	BUILD_ASSERT(DT_INST_PROP(id, green_output_current) <= LP5562_MAX_CURRENT_SETTING,\
-		"Green channel current must be between 0 and 25.5 mA.");	\
-	BUILD_ASSERT(DT_INST_PROP(id, blue_output_current) <= LP5562_MAX_CURRENT_SETTING,\
-		"Blue channel current must be between 0 and 25.5 mA.");	\
-	BUILD_ASSERT(DT_INST_PROP(id, white_output_current) <= LP5562_MAX_CURRENT_SETTING,\
-		"White channel current must be between 0 and 25.5 mA.");	\
-	static const struct lp5562_config lp5562_config_##id = {	\
-		.bus = I2C_DT_SPEC_INST_GET(id),			\
-		.r_current = DT_INST_PROP(id, red_output_current),	\
-		.g_current = DT_INST_PROP(id, green_output_current),	\
-		.b_current = DT_INST_PROP(id, blue_output_current),	\
-		.w_current = DT_INST_PROP(id, white_output_current),	\
-	};								\
-									\
-	struct lp5562_data lp5562_data_##id;				\
-	DEVICE_DT_INST_DEFINE(id, &lp5562_led_init, NULL,		\
-			&lp5562_data_##id,				\
-			&lp5562_config_##id, POST_KERNEL,		\
-			CONFIG_LED_INIT_PRIORITY,			\
-			&lp5562_led_api);				\
+#define LP5562_DEFINE(id)                                                                          \
+	BUILD_ASSERT(DT_INST_PROP(id, red_output_current) <= LP5562_MAX_CURRENT_SETTING,           \
+		     "Red channel current must be between 0 and 25.5 mA.");                        \
+	BUILD_ASSERT(DT_INST_PROP(id, green_output_current) <= LP5562_MAX_CURRENT_SETTING,         \
+		     "Green channel current must be between 0 and 25.5 mA.");                      \
+	BUILD_ASSERT(DT_INST_PROP(id, blue_output_current) <= LP5562_MAX_CURRENT_SETTING,          \
+		     "Blue channel current must be between 0 and 25.5 mA.");                       \
+	BUILD_ASSERT(DT_INST_PROP(id, white_output_current) <= LP5562_MAX_CURRENT_SETTING,         \
+		     "White channel current must be between 0 and 25.5 mA.");                      \
+	static const struct lp5562_config lp5562_config_##id = {                                   \
+		.bus = I2C_DT_SPEC_INST_GET(id),                                                   \
+		.r_current = DT_INST_PROP(id, red_output_current),                                 \
+		.g_current = DT_INST_PROP(id, green_output_current),                               \
+		.b_current = DT_INST_PROP(id, blue_output_current),                                \
+		.w_current = DT_INST_PROP(id, white_output_current),                               \
+		.en_pin = GPIO_DT_SPEC_INST_GET_OR(id, en_gpios, {0}),                             \
+	};                                                                                         \
+                                                                                                   \
+	struct lp5562_data lp5562_data_##id;                                                       \
+	DEVICE_DT_INST_DEFINE(id, &lp5562_led_init, NULL, &lp5562_data_##id, &lp5562_config_##id,  \
+			      POST_KERNEL, CONFIG_LED_INIT_PRIORITY, &lp5562_led_api);
```

Comment:
Are there any changes above related to the ? If it's just a reindentation, please put it in a dedicated commit.

Reply:
Compliance CI warn about formatting of diff to be compatible with clang-format. That's why formatted the diff.
I will omit it.